Content/AbstractContentController.cs

```diff
@@ -19,12 +19,21 @@ public abstract class AbstractContentController<TContent> : AbstractContentContr
         public new TContent Content
         {
             get => (TContent)base.Content;
-            protected set => base.Content = value;
         }
 
         //TODO: Look at replacing the contentGroupID with Enums - https://github.com/scratch-games/anvil-csharp-core/issues/19
+        protected AbstractContentController(string contentGroupID)
+            :base(contentGroupID) { }
+
+        [Obsolete]
```

Comment:
Should we give the mitigation step in this tag? 

"Ensure your derived content controller handles loading the content" etc?